### PR TITLE
Validate values sent by form

### PR DESCRIPTION
**Pull request by Vampouille**
Check that year is numeric and is part of allowed year, check format
against strict list : shp, mifmid, postgis and check that proj param is
numeric.

### CODE REVIEW

File: foncier/app.py [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
 import celery.states as states
 import os
+import logging
 
 # create the app:
 app = Flask(__name__)
 
 ROLE_PREFIX = os.environ.get('ROLE_PREFIX', 'ROLE_FONCIER_')
 DEBUG = os.environ.get('DEBUG', 'False')
+logger = logging.getLogger('worker')
```
Comment (fvanderbiest) [inline foncier/app.py]: `logging.getLogger('app')` ?

**Review by fvanderbiest — CHANGES_REQUESTED** [review]
LGTM, except for the logger name

File: foncier/app.py (hunk 1 above)
Comment (Vampouille) [reply foncier/app.py]: Should be fixed by 1bacc531e25dfce7dfdb8c6a4faf2ab34039fa93

**Comment by fvanderbiest** [comment]
Thanks. Please merge :-)

**Pull request merged**